app/assets/stylesheets/hyrax/_work-show.scss

```diff
@@ -56,3 +56,28 @@ ul.tabular {
   margin-bottom: 18px;
   margin-top: -8px;
 }
+
+.panel-workflow {
+  background: #FFF;
+  border-color: #e0e0e0;
+  left: 0;
```

Comment:
Do we need this?

Reply:
I think so, but can test again. I believe I had some issues without it.